ietf/templates/nomcom/download_questio.txt

```diff
@@ -0,0 +1,9 @@
+{# Copyright The IETF Trust 2023, All Rights Reserved #}{% autoescape off %}{% load nomcom_tags %}Questionnaire response from {{ nominee.person.name }}
+
+From:      {{ feedback.author|formatted_email|default:"Anonymous" }}
+Date:      {{ feedback.time|date:"Y-m-d" }}
+Positions: {{ positions }}{% if feedback.subject %}
```

Comment:
For some reason, `{{ feedback.positions.all|join:"," }} ` renders fine in the html template, but renders as `<PositionQuerySet [<Position: Test Area Director>]>` here, so I moved the join to the view.

Reply:
Curious, likely to be a consequence of render_to_string, but not worth pursuing further at this time.